src/main/java/racingcar/controller/RacingCarWebController.java

```diff
@@ -0,0 +1,45 @@
+package racingcar.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseBody;
+import racingcar.dao.RacingCarGameDao;
+import racingcar.domain.Cars;
+import racingcar.dto.RacingGameRequestDto;
+import racingcar.dto.ResultResponseDto;
+import racingcar.repository.RacingCarRepository;
+import racingcar.service.RacingCarService;
+import racingcar.util.RandomNumberGenerator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Controller
+public class RacingCarWebController {
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    @PostMapping("/plays")
+    @ResponseBody
+    public ResponseEntity<ResultResponseDto> play(@RequestBody RacingGameRequestDto racingGameRequestDto) {
+        List<String> names = Arrays.stream(racingGameRequestDto.getNames().split(","))
+                .map(String::trim)
+                .collect(Collectors.toList());
+
+        RacingCarService racingCarService = new RacingCarService(
+                Cars.of(names),
+                new RandomNumberGenerator(),
+                new RacingCarRepository(new RacingCarGameDao(jdbcTemplate))
+        );
+        racingCarService.play(racingGameRequestDto.getCount());
+
+        ResultResponseDto resultResponseDto = new ResultResponseDto(racingCarService.getWinners(), racingCarService.getCars());
+        return ResponseEntity.ok().body(resultResponseDto);
```

Comment:
```java
	/**
	 * A shortcut for creating a {@code ResponseEntity} with the given body
	 * and the status set to {@linkplain HttpStatus#OK OK}.
	 * @param body the body of the response entity (possibly empty)
	 * @return the created {@code ResponseEntity}
	 * @since 4.1
	 */
	public static <T> ResponseEntity<T> ok(@Nullable T body) {
		return ok().body(body);
	}
```
를 사용할 수도 있습니다!

Reply:
조금더 간결하게 코드를 관리할 수 있는 방법이 있었네요!